corehq/apps/reports/models.py

```diff
@@ -939,17 +939,6 @@ def uses_cases(self):
         return False
 
 
-class FormDeidExportSchema(FormExportSchema):
```

Comment:
I think I committed this in the middle of something and then forgot about it. There's only one commit that ever mentions `FormDeidExportSchema` in the diff:
```
$ gl -S FormDeidExportSchema
commit fdd53981949676d0da2a00517b2f92e8bef49637
Author: Daniel Roberts <droberts@dimagi.com>
Date:   Fri May 11 11:26:06 2012 -0400

    toying with idea of subclassing FormExportSchema
```

https://github.com/dimagi/commcare-hq/commit/fdd53981949676d0da2a00517b2f92e8bef49637. This was before PR reviews, back when we were all pushing to master; clearly "toying with [the] idea" of something was enough to get it into master at the time...